ext/rorocos/rorocos.cc

```diff
@@ -219,7 +219,11 @@ static VALUE task_context_attribute_names(VALUE self)
         corba_blocking_fct_call_with_result(bind(&_objref_CConfigurationInterface::getAttributeList,(_objref_CConfigurationInterface*)context.main_service));
     for (unsigned int i = 0; i != names->length(); ++i)
     {
-        CORBA::String_var name = names[i];
+	#if RTT_VERSION_GTE(2,9,0)
+		CORBA::String_var name = names[i].name;
```

Comment:
Bogus indentation. You use 4-spaces tabs and 8 is the standard. Indent with spaces.